makeflow/src/mountfile.c

```diff
@@ -0,0 +1,660 @@
+/*
+Copyright (C) 2008- The University of Notre Dame
```

Comment:
Okay. I need to do some file renaming to match these new files to the naming style of `makeflow. 
We can talk about moving it to dttools after I finish renaming them.